packages/i18n/src/i18n.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Tannin from 'tannin';
+
+const TEXT_DOMAIN = 'web-stories';
+
+const tannin = new Tannin({
+  [TEXT_DOMAIN]: {
+    '': {
+      plural_forms(n) {
+        return n === 1 ? 0 : 1;
+      },
+    },
```

Comment:
Can you explain this?

Reply:
This is just the default configuration needed for Jed / Tannin